README.md

```diff
@@ -87,9 +99,34 @@ pack install-app katla
 If you no longer require *katla* and want to remove it, run
 
 ```sh
-pack remove katla
+pack remove-app katla
 ```
 
+### Updating Idris2, pack, and packages
+
+To update pack's Idris2 installation:
+
+```sh
+pack switch latest
+```
+
+Note that because packages are tied to Idris2 versions, if you update
+Idris2 then you may need to update your packages too.
+
+To update pack itself:
+
+```sh
+pack update
+```
+
+To update your installed packages:
+
+```sh
+pack update-db
```

Comment:
This does not update the installed packages but only the list of available pack collections, so this section should be removed.